### PR TITLE
[SPARK-26497][DOCS][K8S] Show users where the pre-packaged SparkR and PySpark Dockerfiles

### PR DESCRIPTION
**Pull request by holdenk**
## What changes were proposed in this pull request?

Show users where the pre-packaged SparkR and PySpark Dockerfiles
## How was this patch tested?

Run script manually verify output.

### CODE REVIEW

File: bin/docker-image-tool.sh [hunk 1]
```diff
@@ -232,7 +236,7 @@ Examples:
     $0 -m -t testing build
 
   - Build PySpark docker image
-    $0 -r docker.io/myrepo -t v2.3.0 -p kubernetes/dockerfiles/spark/bindings/python/Dockerfile build
+    $0 -r docker.io/myrepo -t v2.3.0 -p resource-managers/kubernetes/docker/src/main/dockerfiles/spark/bindings/python/Dockerfile
```
[the remark this answers is not in the document]
Comment (holdenk) [reply bin/docker-image-tool.sh]: Good catch, unintended change I'll fix.

**Comment by felixcheung** [comment]
LGTM

File: bin/docker-image-tool.sh [hunk 2]
```diff
@@ -208,8 +208,16 @@ Options:
   -f file               Dockerfile to build for JVM based Jobs. By default builds the Dockerfile shipped with Spark.
   -p file               (Optional) Dockerfile to build for PySpark Jobs. Builds Python dependencies and ships with Spark.
                         Skips building PySpark docker image if not specified.
+                        A sample PySpark Dockerfile is at:
+                        kubernetes/dockerfiles/spark/bindings/python/Dockerfile
+			File must be relative the build path, in dev this is copied from:
+			resource-managers/kubernetes/docker/src/main/dockerfiles
   -R file               (Optional) Dockerfile to build for SparkR Jobs. Builds R dependencies and ships with Spark.
                         Skips building SparkR docker image if not specified.
+			 A sample SparkR Dockerfile is at:
```
Comment (felixcheung) [inline bin/docker-image-tool.sh]: ` A` extra space

Comment (felixcheung) [reply bin/docker-image-tool.sh]: it looks like the preview shows something different. perhaps space vs tab is shown differently.

**Comment by mccheah** [comment]
@holdenk want to update the formatting issues from https://github.com/apache/spark/pull/23399#discussion_r244547474 and then we can merge?

File: bin/docker-image-tool.sh [hunk 3]
```diff
@@ -219,6 +227,8 @@ Options:
   -b arg                Build arg to build or push the image. For multiple build args, this option needs to
                         be used separately for each build arg.
 
+In dev the build enviroment copies in resource-managers/kubernetes/docker/src/main/dockerfiles
```
Comment (srowen) [inline bin/docker-image-tool.sh]: Nit: does this need a little clarification ... does it mean the build environment copies something from A to B, or example copies are here?

File: bin/docker-image-tool.sh [hunk 4]
```diff
@@ -208,8 +208,16 @@ Options:
   -f file               Dockerfile to build for JVM based Jobs. By default builds the Dockerfile shipped with Spark.
   -p file               (Optional) Dockerfile to build for PySpark Jobs. Builds Python dependencies and ships with Spark.
                         Skips building PySpark docker image if not specified.
+                        A sample PySpark Dockerfile is at:
+                        kubernetes/dockerfiles/spark/bindings/python/Dockerfile
+                        File must be relative the build path, in dev this is copied from:
```
Comment (srowen) [inline bin/docker-image-tool.sh]: Super nit: ; or . instead of the , here

**Review by srowen — COMMENTED** [review]
Seems fine to add

File: bin/docker-image-tool.sh (hunk 3 above)
Comment (vanzin) [reply bin/docker-image-tool.sh]: Yeah, it seems something is missing from this sentence.

**Comment by srowen** [comment]
@holdenk I think it's OK to merge pending a few tiny comments above

**Comment by vanzin** [comment]
@holdenk ping?

**Comment by srowen** [comment]
@holdenk do you want to take another look at this one?

**Comment by srowen** [comment]
Feel free to reopen with further changes

**Pull request closed without merging**